### PR TITLE
Hotfix remove currency validator in create donation

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #1218

### CODE REVIEW

File: src/utils/validators/graphqlQueryValidators.ts [hunk 1]
```diff
@@ -84,15 +83,7 @@ export const createDonationQueryValidator = Joi.object({
     .required()
     .valid(...Object.values(NETWORK_IDS)),
   tokenAddress: Joi.string().pattern(ethereumWalletAddressRegex),
-  token: Joi.string()
-    .required()
-    .pattern(tokenSymbolRegex)
```
Comment (aminlatifi) [inline src/utils/validators/graphqlQueryValidators.ts:89]: Just remove the pattern matching and please keep the error message

File: src/utils/validators/graphqlQueryValidators.ts [hunk 2]
```diff
@@ -18,7 +18,6 @@ const resourcePerDateRegex = new RegExp(
 
 const ethereumWalletAddressRegex = /^0x[a-fA-F0-9]{40}$/;
 const txHashRegex = /^0x[a-fA-F0-9]{64}$/;
-const tokenSymbolRegex = /^[a-zA-Z0-9]{2,10}$/; // OPTIMISTIC OP token is 2 chars long
```
Comment (aminlatifi) [inline src/utils/validators/graphqlQueryValidators.ts:21]: Keep it as commented

File: src/utils/validators/graphqlQueryValidators.ts (hunk 1 above)
Comment (aminlatifi) [reply src/utils/validators/graphqlQueryValidators.ts:89]: Or keep it as commented!

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/utils/validators/graphqlQueryValidators.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply src/utils/validators/graphqlQueryValidators.ts:89]: Done

File: src/utils/validators/graphqlQueryValidators.ts (hunk 2 above)
Comment (mohammadranjbarz) [reply src/utils/validators/graphqlQueryValidators.ts:21]: Done

File: src/utils/validators/graphqlQueryValidators.ts (hunk 1 above)
Comment (aminlatifi) [reply src/utils/validators/graphqlQueryValidators.ts:89]: Sorry, I didn't mean commenting out the error message!

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Pull request merged**